File: codesignal/arcade/Intro/Csharp/commonCharacterCount.cs
int commonCharacterCount(string s1, string s2) {
    int[] asciiLettersS1 = new int[26];
    int[] asciiLettersS2 = new int[26];
    int ascii = 0;
    int count = 0;
    for(int i = 0; i < s1.Length; i++){
        ascii = (int)s1[i] - 97;
        
        if(asciiLettersS1[ascii] == 0){
            asciiLettersS1[ascii] = 1;
        }else{
            asciiLettersS1[ascii] += 1;
        }
        
    }
    
    for(int i = 0; i < s2.Length; i++){
        ascii = (int)s2[i] - 97;
        
        if(asciiLettersS2[ascii] == 0){
            asciiLettersS2[ascii] = 1;
        }else{
            asciiLettersS2[ascii] += 1;
        }
    }
    
    for(int i = 0; i < asciiLettersS1.Length; i++){
        count += Math.Min(asciiLettersS1[i], asciiLettersS2[i]);
    }
    
    return count;
}

File: codesignal/arcade/Intro/JS/commonCharacterCount.js
function commonCharacterCount(s1, s2) {
    let count = 0
    let asciiLettersS1 = new Array(26).fill(0)
    let asciiLettersS2 = new Array(26).fill(0)
    let ascii = 0
    
    for(let i = 0; i < s1.length; i++){
        ascii = s1[i].charCodeAt(0) - 97
        
        asciiLettersS1[ascii] == 0 ? asciiLettersS1[ascii] = 1 : asciiLettersS1[ascii] += 1 
    }
    
    for(let i = 0; i < s2.length; i++){
        ascii = s2[i].charCodeAt(0) - 97
        
        asciiLettersS2[ascii] == 0 ? asciiLettersS2[ascii] = 1 : asciiLettersS2[ascii] += 1 
    }
    
    for(let i = 0; i < asciiLettersS1.length; i++){
        count += Math.min(asciiLettersS1[i], asciiLettersS2[i])
    }
    return count
}
